src/Utils/sortpermFast.jl

```diff
@@ -3,15 +3,11 @@ export sortpermFast
 
 
 function sortpermFast(A)
-	const n = length(A)
-   ii = Array(Int64,n)
-
-   for i = 1:n
-   	ii[i] = i
-   end
+   const n = length(A)
 
+   ii = collect(1:n)
    B = copy(A)
-   quicksort!(B,ii)
+   quicksort!(B,ii, 1,n)
```

Comment:
Don't think it needs to hold up this PR but I'm curious if Roman's implementation of quicksort beats the `sort` function from the Julia standard library. Maybe it does but if not, this code should probably not be part of jInv.